File: src/main/java/org/radarbase/management/security/jwt/SignatureException.java
package org.radarbase.management.security.jwt;

public class SignatureException extends RuntimeException {

    public SignatureException() {
        super();
    }

    public SignatureException(String message) {
        super(message);
    }

    public SignatureException(String message, Throwable ex) {
        super(message, ex);
    }

    public SignatureException(Throwable ex) {
        super(ex);
    }
}

File: src/main/java/org/radarbase/management/security/jwt/SignatureException.kt
package org.radarbase.management.security.jwt

class SignatureException : RuntimeException {
    constructor() : super()
    constructor(message: String?) : super(message)
    constructor(message: String?, ex: Throwable?) : super(message, ex)
    constructor(ex: Throwable?) : super(ex)
}
